src/core/toga/widgets/splitcontainer.py

```diff
@@ -59,14 +59,14 @@ def content(self, content):
             widget._update_layout()
             widget.app = self.app
             widget.window = self.window
-
+            # Fixme move all container stuff to _impl level.
```

Comment:
I moved the container stuff to _impl level. But I didn't managed to set the min_width/height constraints.